### ISSUE TITLE
Null ref: T2.Creature.FallOver

### ISSUE THREAD

**Issue by ChrisNZL** [ChrisNZL/Tallowmere2#492]
Auto bug report from 0.1.3.

Feedback IDs:
- 20200422-7SPXZ
- 20200422-MWCTT

```
12:20:40, Frame 292736, EXCEPTION »  NullReferenceException: Object reference not set to an instance of an object
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

T2.Creature.FallOver ()
T2.Creature.InstantiateCreature (T2.CreatureStruct creatureStruct, UnityEngine.Transform container, T2.SeedFactory seedFactory, System.Boolean allowAI)
T2.DungeonRoom.Update ()

GameStates: InitialisingAsNetworkHost, RompingThroughDungeon, TransitioningToDungeonRoom
GameSetupMode: NetworkCoop
Players: 2
Room: 4 / 4
RoomModifiers: None
```

**Comment by ChrisNZL** [comment]
Additional Feedback ID: 20200503-5CPBN

Different game states from above: 

```
12:42:26, Frame 708, LOG »  RELAY CLIENT: Connecting to server...
12:42:27, Frame 751, LOG »  NETWORK CLIENT: Connecting to X.X.X.X ...
12:42:27, Frame 754, LOG »  NETWORK CLIENT: RelayClient connected to server X.X.X.X

12:42:28, Frame 823, EXCEPTION »  NullReferenceException: Object reference not set to an instance of an object
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

T2.Creature.FallOver ()
T2.Creature.InstantiateCreature (T2.CreatureStruct creatureStruct, UnityEngine.Transform container, T2.SeedFactory seedFactory, System.Boolean allowAI)
T2.DungeonRoom.Update ()

GameStates: ShowingTitleScreen, TransitioningToDungeonRoom
```

**Comment by ChrisNZL** [comment]
v0.1.4, happening in single-player too.

Feedback ID: 20200610-5XDHN

```
2:53:41, Frame 21888, EXCEPTION »  NullReferenceException: Object reference not set to an instance of an object
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

T2.Creature.FallOver ()
T2.Creature.InstantiateCreature (T2.CreatureStruct creatureStruct, UnityEngine.Transform container, T2.SeedFactory seedFactory, System.Boolean allowAI)
T2.DungeonRoom.Update ()

GameStates: RompingThroughDungeon, UsingWaypoint, TransitioningToDungeonRoom
GameSetupMode: Singleplayer
Players: 1
DungeonRoomTransitionerState: WaitingForRoomToLoad
Room: 2 / 2
RoomModifiers: None
YouHaveDiedState: Null
```

**Comment by ChrisNZL** [comment]
Should be fixed in 0.1.4a now.
